CHANGELOG.md

```diff
@@ -60,6 +60,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 - TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
 - **General**: Add --ca-dir flag to KEDA operator to specify directories with CA certificates for scalers to authenticate TLS connections (defaults to /custom/ca) ([#5860](https://github.com/kedacore/keda/issues/5860))
+- **General**: Add the generateEmbeddedObjectMeta flag when generating CRD ([#5908](https://github.com/kedacore/keda/issues/5908))
```

Comment:
We may want to elaborate on what this means for end-users though? Why does this matter?